File: pet-clinic-java/src/main/java/com/github/rskupnik/petclinic/PetClinicApplication.java
package com.github.rskupnik.petclinic;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.EnableWebMvc;

@SpringBootApplication
public class PetClinicApplication {

	public static void main(String[] args) {
		SpringApplication.run(PetClinicApplication.class, args);
	}
}

File: pet-clinic-scala/src/main/scala/com/github/rskupnik/PetClinicApplication.scala
package com.github.rskupnik

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.context.annotation.{ComponentScan, Configuration}

@Configuration
@EnableAutoConfiguration
@ComponentScan
class PetClinicConfig

object PetClinicApplication extends App {
  SpringApplication.run(classOf[PetClinicConfig])
}
